faros_event.sh

```diff
@@ -22,13 +22,14 @@ FAROS_GRAPH_DEFAULT="default"
 FAROS_URL_DEFAULT="https://prod.api.faros.ai"
 FAROS_ORIGIN_DEFAULT="Faros_Script_Event"
 FAROS_SOURCE_DEFAULT="Faros_Script"
-FAROS_APP_PLATFORM_DEFAULT="NA"
+FAROS_APP_PLATFORM_DEFAULT=""
```

Comment:
Shubha requested that I change this default to `""` so I just included it as it also required a readme change.